processor/src/main/java/org/mapstruct/ap/internal/conversion/BigDecimalToStringConversion.java

```diff
@@ -19,15 +19,16 @@
 package org.mapstruct.ap.internal.conversion;
 
```

Comment:
There isn't an empty line between the header and the package declaration anywhere, as far as I could see, so I thought we should leave it as it is.

This is more about the missing empty lines between method declarations (generated code and the processor code), and those white-outs in some files (there was one spot with five empty lines).